### ISSUE TITLE
is this code  can be used for Biomedical image segmentation.viz only two classes  

### ISSUE THREAD

**Issue by rddvsr** [legokichi/keras-segnet#2]

**Comment by legokichi** [comment]
For that purpose [u-net](https://github.com/legokichi/keras-pix2pix/blob/master/model_unet.py) is better than segnet. I have never tried this keras-segnet code outside the CamVid dataset.
Try using the [dice_coef loss function](https://github.com/legokichi/keras-pix2pix/blob/master/train.py#L41) with 2 output values of u-net.
